README.rst

```diff
@@ -48,20 +48,32 @@ Adjust according to your operating system or personal preferences.
 #. Install a virtual environment for development (Python 2 or Python 3, you choose!)::
 
     $ sudo pip install virtualenvwrapper
+
+#. Follow the instructions from http://virtualenvwrapper.readthedocs.org/en/latest/install.html#basic-installation.
+   You will need to source the virtualenvwrapper.sh file for the following commands to work::
+
     $ mkvirtualenv --python=python3 kolibri
     $ workon kolibri
 
+#. Install the other packages needed for development::
+
+   $ sudo pip install -r requirements/dev.txt
```

Comment:
Any reason why we need to do a `sudo pip install...` here?  I think we can only do a `pip install...` because a virtualenv is already activated.